Assets/Scripts/CodeBlockFamily.cs

```diff
@@ -50,16 +60,17 @@ public class CodeBlockFamily : MonoBehaviour {
         }
 
         private void SetBlockPosition(GameObject block, int index) {
-            block.transform.position = new Vector3(
+            block.transform.SnapToParent(transform);
+            block.transform.localScale = Vector3.one;
+            block.transform.localPosition = new Vector3(
                     shelfMeshRenderer.bounds.center.x,
-                    shelfMeshRenderer.bounds.extents.y - 0.2f - index * shelfMeshRenderer.bounds.size.y / (blocksInFamily.Count + 0.25f),
-                    transform.position.z);
+                    shelfMeshRenderer.bounds.extents.y - 0.2f - (index * shelfMeshRenderer.bounds.size.y * 4f) / (blocksInFamily.Count + 0.25f),
```

Comment:
could you turn these `4f` etc floats into variables with names that are private members of this class?

Reply:
this makes it a bit easier to understand